### PR TITLE
Fix Object.defineProperty

### PR DESCRIPTION
**Pull request by marcioj**

### CODE REVIEW

**Comment by marcioj** [comment]
//cc @rwjblue @stefanpenner

File: packages/ember-metal/lib/properties.js [hunk 1]
```diff
@@ -136,10 +136,8 @@ export function defineProperty(obj, keyName, desc, data, meta) {
         obj[keyName] = data;
       }
     } else {
-      value = desc;
```
Comment (mixonic) [inline packages/ember-metal/lib/properties.js]: `value` here is being set for `didDefineProperty a few lines below. Seems like we have no tests, but perhaps it should not be removed?

Comment (marcioj) [inline packages/ember-metal/lib/properties.js]: You're right. I reverted this line and added a test to `didDefineProperty` hook like the other tests. Thanks!

Comment (rwjblue) [inline packages/ember-metal/lib/properties.js]: You rock, thank you!

**Comment by stefanpenner** [comment]
Scary that no tests caught that one :p

**Comment by stefanpenner** [comment]
thanks !

**Comment by marcioj** [comment]
Congratulations for reaching 10.000 commits. You guys are the best!

**Comment by stefanpenner** [comment]
> Congratulations for reaching 10.000 commits. You guys are the best!

thanks for being 10,000th

**Pull request merged**